firestore/integration_test_internal/src/aggregate_query_snapshot_test.cc

```diff
@@ -0,0 +1,283 @@
+/*
```

Comment:
All of the comments for aggregate_query_test.cc also apply to this file, aggregate_query_snapshot_test.cc.